modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/AlphaDBImport_conf.pm

```diff
@@ -72,6 +72,9 @@ sub default_options {
         user         => 'ensadmin',
         alphafold_data_file => '/nfs/ftp/public/databases/alphafold/accession_ids.csv',
         uniparc_data_file => '/nfs/ftp/public/databases/uniprot/current_release/knowledgebase/idmapping/idmapping_selected.tab.gz',
+        # Do not blindly increase the version here. GIFTS data is seldom
+        # refreshed - the current version probably has no GIFTS data.
+        gifts_dir    => '/nfs/production/flicek/ensembl/production/alphafold_gifts/107',
```

Comment:
I would prefer this path to be a parameter, with no default, and we set it in the SOP. so we'll know when to change it/override it. and this way no way to launch the pipeline with older data than expected because we forgot the parameter. Or may just be the "gift_version" to be mandatory, if you prefer.